engine/util/src/main/scala/pl/touk/nussknacker/engine/util/namespaces/ObjectNamingProvider.scala

```diff
@@ -0,0 +1,18 @@
+package pl.touk.nussknacker.engine.util.namespaces
```

Comment:
I think it shoul be in api.

Reply:
It can't be without introducing a cyclic dependency. `ObjectNamingProvider` uses `ScalaServiceLoader` which is a member of `util` package - which, in turn, depends on `api` package.

Reply:
But `ObjectNaming` can be in api? It make sens that `ObjectNaming` is part of an api and we have provider for them in utils...

Reply:
Yes, @arkadius, that's what I did.

Reply:
Indeed, my mistake